code/game/objects/effects/spiders.dm

```diff
@@ -154,7 +154,7 @@
 	var/amount_grown = 0
 	var/obj/machinery/atmospherics/unary/vent_pump/entry_vent
 	var/travelling_in_vent = 0
-	var/list/grow_as = list(/mob/living/simple_mob/animal/giant_spider, /mob/living/simple_mob/animal/giant_spider/nurse, /mob/living/simple_mob/animal/giant_spider/hunter)

```

Comment:
Those eggs aren't used normally, so them spawning a nurse is fine I believe. If they are mapped in or used somewhere, I think it's better to make a subtype that spawns only hunters and regulars instead.